### PR TITLE
Feat: Landing page

### PR DESCRIPTION
**Pull request by nishasen**
Added landing page. 
It has different route links, user can go to product page by using house paths or category paths.
image
image

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
Looks Good and Nice UI

File: diagon-alley/src/Components/HeroImage/HeroImage.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button } from '..';
```
Comment (shubhamkumar648) [inline diagon-alley/src/Components/HeroImage/HeroImage.jsx:3]: I think button component not import properly

**Review by shubhamkumar648 — APPROVED** [review]
Nice structure of the code in two small components!

File: diagon-alley/src/Components/HeroImage/HeroImage.jsx (hunk 1 above)
Comment (nishasen) [reply diagon-alley/src/Components/HeroImage/HeroImage.jsx:3]: Its imported from index of components and hereImage is also in components so its that way

**Pull request merged**